source/_posts/bars-to-code-at.md

```diff
@@ -0,0 +1,134 @@
+wobbly little cafe table.
+
+Second, bars tend to have more power outlets. Most recently-opened bars have
+power strips running under the bar or along seating areas. I think this is
+because bars want people to linger, and coffee shops don't. Someone staying at a
+bar for a few hours will likely order a number of expensive drinks, where
+someone staying at a cafe will only order a coffee and maybe a snack.
+
+Third, bars tend to be darker and cooler, relative to cafes, which are often
+stuffy and sunny.
+
+Finally, after a good coding session, I often find myself in a sort of distant,
+aphasic state. Everything seems flat and it's hard to think of anything to say.
+Being at a bar gives me a chance to turn off my computer, find someone to talk
+to, and reintegrate.
+
+Of course, not all bars are good places to code. Some bars are problematic for
+social reasons, and others are problematic for practical reasons. Here's a list
+of bars that Computer Lab members have found suitable for computer use.
+
+
+### Skytown
+Bushwick [Map](https://goo.gl/maps/SwAZJAFYTwR2)
+
+[skytownbk.com](http://www.skytownbk.com/)
+
+Skytown is my personal favorite on this list, probably for nostalgic reasons. It
+was the first bar that I coded at. In my time there, I've never found the place
+unpleasantly loud or crowded, and the music tends to be interesting enough that
+I'm often able to forego use of my [Bose QuietComfort 35 Noise Cancelling
+Headphones](https://www.amazon.com/Bose-QuietComfort-Wireless-Headphones-Cancelling/dp/B01E3SNO1G).
+
+Skytown serves cheap and excellent bean tacos until 2:00AM, which is great for
+people who like to [start coding at
+11PM](https://twitter.com/mannynotfound/status/862147137769005056). There's
+power outlets under the bar (sometimes you have to ask the bartenders to plug in
+the strip), and stable wifi. Finally, I've met a wide range of interesting
+people there, including a recruiter from the Fed who attempted to whiteboard me
+on the spot.
+
+### Sugarburg
+Williamsburg [Map](https://goo.gl/maps/A2iQNHNmyTs)
+
+[sugarburg.com](http://sugarburg.com)
+
+Sugarburg features great natural light, unconventional seating, above-par bar
+food, and good service. The vertically slatted windows make this a good place to
+enjoy the weather during the spring and summer. The bartenders encourage people
+to use the bar as a workspace during the day, making Sugarburg a good place to
+work or meet regularly. The bar is large and Bourbun-oriented, making Sugarburg
+a great place to hang out late into the night.
+
+### Boobie Trap
+Bushwick [Map](https://goo.gl/maps/Q3CUv7sTRVE2)
+
+[boobietrapbrooklyn.com](http://boobietrapbrooklyn.com)
+
+[Robert Jensen](http://you.could.use.some.r1b.solutions/) says that Boobie Trap
+is actually a pretty horrible place to code (it's very loud and the tables are
+sticky), but it's one of the few places along the L in Bushwick with good
+internet.  He goes there to push docker containers and backups.
```

Comment:
Mention that they have a dedicated charging station for phones & a power strip next to that.